contracts/staking/contracts/src/fees/MixinExchangeFees.sol

```diff
@@ -217,8 +263,9 @@ contract MixinExchangeFees is
             );
 
             // store pool stats
+            uint256 protocolFeeBalance = protocolFeesThisEpochByPool[poolId];
```

Comment:
Is this an optimization?

Reply:
It looks like it actually makes it less efficient. I think that it's a holdover from how it would have worked before WETH protocol fees.